libqtile/widget/nvidia_sensors.py

```diff
@@ -0,0 +1,64 @@
+import csv
+import re
+
+from libqtile.widget import base
+
+sensors_mapping = {
+    'fan_speed': 'fan.speed',
+    'perf': 'pstate',
+    'temp': 'temperature.gpu',
+}
+
+
+class NvidiaSensors(base.InLoopPollText):
```

Comment:
Might be worth making this part of the threadpool, as doing it synchronously will hang the WM if the command ever hangs.